frontend/js/src/common/brainzplayer/YoutubePlayer.tsx

```diff
@@ -396,6 +407,13 @@ export default class YoutubePlayer extends React.Component<YoutubePlayerProps>
           <button className="btn btn-sm youtube-drag-handle" type="button">
             <FontAwesomeIcon icon={faArrowsAlt} />
           </button>
+          <button
+            className="btn btn-sm youtube-button"
+            type="button"
+            onClick={onClose}
```

Comment:
And finally, this will now use `handleHide`:
```suggestion
            onClick={handleHide}
```